recipes/jaeger/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set goname = "github.com/jaegertracing/jaeger" %}
+{% set version = "1.14.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # We must use git here beause the program looks
+  # for the git sha&tag to determine its version
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/v{{version}}.tar.gz
+    sha256: 65c39e068cfb23f67529b75617202adca5ab41668942ae91fee14056b7657620 
+
+
+build:
+  number: 0
+  script:
+    - set -x
+    - pushd {{ pkg_src }}
+    - git submodule update --init --recursive
+    - dep ensure
+    - make install-tools
+    - make build-ui
+    - make build-bbinaries-darwin  # [osx]
+    - make build-binaries-linux  # [linux]
+    - make build-binaries-windows  # [win]
+    - "mkdir -vp ${PREFIX}/bin"
+    - "cp -v cmd/all-in-one/all-in-one-* ${PREFIX}/bin/all-in-one"
+    - "cp -v cmd/agent/agent-* ${PREFIX}/bin/agent"
+    - "cp -v cmd/query/query-* ${PREFIX}/bin/query"
+    - "cp -v cmd/collector/collector-* ${PREFIX}/bin/collector"
+    - "cp -v cmd/ingester/ingester-* ${PREFIX}/bin/ingester"
+    - "cp -v examples/hotrod/hotrod-* ${PREFIX}/bin/hotrod"
+    - "chmod -v 755 ${PREFIX}/bin/*"
+
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - go-dep
+    - nodejs
+    - yarn
```

Comment:
yarn, node and go seem like host deps to me

Reply:
TBH not sure what the difference is between host deps and these. 

You don't need them once you build the binary, you just need it to build it.

Reply:
build is the OS that the build occurs on (where the compilers are). Host is the OS target (where binaries will be deployed to). This is an especially important distinction for cross-compiling.